packages/kbn-build/src/commands/start.js

```diff
@@ -0,0 +1,53 @@
+import chalk from 'chalk';
+
+import { getPackages, topologicallyBatchPackages } from '../utils/packages';
+
+export const name = 'start';
+export const description = 'Start Kibana and watch packages';
+
+export async function run(config) {
+  const rootPath = config.rootPath;
+  const packagesPaths = config.packages;
+
+  const packages = await getPackages(rootPath, packagesPaths);
+
+  console.log(chalk.bold(`Found [${chalk.green(packages.size)}] packages:\n`));
+  for (const pkg of packages.values()) {
+    console.log(`- ${pkg.name} (${pkg.path})`);
+  }
+
+  // We know we want to start Kibana last, so this is just making _sure_ it's
+  // actually the last package to start.
+  const packagesLessKibana = new Map(packages);
```

Comment:
__question:__ can't we just `const kibana = packages.get('kibana')` and then `packages.delete('kibana')` instead of creating a new map?

Reply:
Yeah, we could. I think both are equally good solutions in this context, so I'll just keep it as-is.